readme.md

```diff
@@ -16,19 +16,30 @@ $ source .venv/bin/activate
 (.venv) $ pip3 install -r requirements.txt
 ```
 
-Note: To step out of the virtual environment, run `$ deactivate`.
+If you will finish working in virtual environment you can step out of it:
 
-## Build it
+```console
+(.venv) $ deactivate
+```
 
+## Build it
+To build project you need to be in virtual environment.
 ```console
 (.venv) $ make html
 ```
 
-## View it
+Sometimes if you make changes in various places at the same time compiler may keep
+complaining about some non existing error. In this case full project rebuild may
+be needed as documents are usually linked to each others:
```

Comment:
My suggestion would be `Some changes may require a full project rebuild`

Reply:
Fixed

Reply:
s/others/other/

Reply:
Fixed